crates/viewer/re_view_spatial/src/ui.rs

```diff
@@ -153,6 +153,12 @@ impl SpatialViewState {
                 self.state_3d.set_spin(spin);
             }
         }
+
+        let mut tracked_entity_path = String::from("None");
+        if let Some(tracked_entity) = self.state_3d.tracked_entity.clone() {
+            tracked_entity_path = tracked_entity.to_string();
+        }
+        ui.label(format!("Tracked entity: {tracked_entity_path}"));
```

Comment:
let's use `ui.horizontal` and put the label next to a "standard entity button" via `item_ui::entity_path_button`  if there is one

nice to have that existing property exposed btw! 👍